files/helpers.py

```diff
@@ -366,18 +369,31 @@ def media_file_info(input_file):
         stdout = run_command(cmd).get("out")
         stream_size = sum([int(line) for line in stdout.split("\n") if line != ""])
         video_bitrate = round((stream_size * 8 / 1024.0) / video_duration, 2)
+    
+    if "r_frame_rate" in video_info.keys():   
+        video_frame_rate = video_info["r_frame_rate"].partition("/")
+        video_frame_rate_n =  video_frame_rate[0]
+        video_frame_rate_d =  video_frame_rate[2]  
 
     ret = {
         "filename": input_file,
         "file_size": file_size,
-        "video_duration": video_duration,
-        "video_frame_rate": float(Fraction(video_info["r_frame_rate"])),
+        "video_duration": video_duration, 
+        "video_frame_rate_n": video_frame_rate_n,
+        "video_frame_rate_d": video_frame_rate_d,
         "video_bitrate": video_bitrate,
         "video_width": video_info["width"],
         "video_height": video_info["height"],
         "video_codec": video_info["codec_name"],
         "has_video": has_video,
         "has_audio": has_audio,
+        "color_range": video_info["color_range"],
```

Comment:
I've uploaded a number of files and some of them (or all) break due to this. Retrieve more defensively all these variables , eg 

```
"color_space": video_info.get("color_space")
```

Are they going to be used somewhere?

Reply:
Yes, it will, for example for HDR->SDR conversion or conversion to Rec.709, but you are right it should be with get(). Should I fix it or can you do it?

Reply:
since this is your PR, it's better if you fix them! thanks

Reply:
Should be fixed now.